sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1860,19 +1860,26 @@ export class ApplicationService extends RecordDataModelService<Application> {
 
   /**
    * Get previous application versions for an application.
-   * @param parentApplicationId parent application id.
+   * @param applicationId application id.
    * @returns previous application versions.
    */
   async getPreviousApplicationVersions(
-    parentApplicationId: number,
+    applicationId: number,
```

Comment:
Minor, I would suggest to have the method renamed to `getApplicationVersions`, not a blocker.

Reply:
I will address that in the PR towards main.